test/com/ableton/PyenvTest.groovy

```diff
@@ -130,15 +130,16 @@ class PyenvTest extends BasePipelineTest {
   void createVirtualEnvUnsupportedPythonVersion() {
     String pythonVersion = '6.6.6'
     String pyenvRoot = '/mock/pyenv/root'
+    boolean errorCalled = false
     helper.registerAllowedMethod('error', [String]) { errorCalled = true }
     helper.registerAllowedMethod('fileExists', [String]) { return true }
     helper.registerAllowedMethod('isUnix', []) { return true }
     helper.addShMock("${pyenvRoot}/bin/pyenv --version", 'pyenv 1.2.3', 0)
-    helper.addShMock(installCommands(pyenvRoot, pythonVersion), '', 1)
+    helper.addShMock(installCommands(pyenvRoot, pythonVersion), '', 0)
```

Comment:
In 43f419a244c71b1e739dde2eb8509c499ecfc2f5, this mock is never called, so I don't see the point of changing its return value. It should just be removed.

On the other hand, there should be a mock for `pyenv install --list`, since it does get called.